src/components/button/button.svelte

```diff
@@ -156,6 +157,7 @@
 
     .content {
       padding: 0 var(--icon-gap);
+      flex-shrink: 0;
```

Comment:
Should this be on `.fab` too? I feel like we want our buttons to be shrinkable

Reply:
> I feel like we want our buttons to be shrinkable

Do you mean we do or don't the `fab` buttons to shrink? `flex-shrink: 0` will prevent them from shrinking.

I wouldn't think we do since they have defined sizes.

Reply:
Sorry, I meant the opposite - should that only be on the `fab` rather than on the normal buttons too.

like, should it be part of the `.fab` class :laughing: sorry for the ambiguity